selfdrive/thermald/thermald.py

```diff
@@ -187,7 +184,7 @@ def thermald_thread():
   has_relay = False
 
   params = Params()
-  pm = PowerMonitoring()
+  pm = PowerMonitoring(params)
```

Comment:
No need to pass the params in. The convention is just to create a new params instance on class init.